cmds/ls/fileinfo_linux.go

```diff
@@ -125,4 +137,9 @@ func (fi longStringer) String() string {
 		fi.size,
 		fi.modTime.Format("Jan _2 15:04"),
 		fi.comp.String())
+
+	if fi.mode&os.ModeType == os.ModeSymlink {
```

Comment:
Oh right you can't. The nil value of a string is "". But the symlink might have the value "". Best to check the ground truth.